.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules/
-
+.pnpm-store/
```

Comment:
You need to put `.pnpm-store/` in local git ignore too.

pnpm should use global pnpm store (to re-use dependencies between projects). Why do you have local `.pnpm-store`?